src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -844,7 +852,7 @@ private static void ApplyFormattingToAnswers(List<Answer> answers, Page page)
 
         private static IEnumerable<Question> PageContainsCheckBoxListQuestions(Page page)
         {
-            return page.Questions.Where(q => q.Input.Type == "CheckBoxList");
+            return page.Questions.Where(q => q.Input.Type == "CheckBoxList" || q.Input.Type=="ComplexCheckBoxList");
```

Comment:
Could this be simplified to .Contains("CheckBoxList") ?

Reply:
Please make it explicit

Reply:
Nope. It should stay as being an explicit list. New input types will appear over time and doing a contains may introduce dodgy behaviour.

Imagine for example 'ComplexRadioWithCheckBoxList"